### PR TITLE
added CRD kraken-cluster to helm chart

### PR DESCRIPTION
**Pull request by leahnp**
moved deployment yaml from artifacts to assets

fulfills RS-112

### CODE REVIEW

File: charts/cluster-controller/values.yaml [hunk 1]
```diff
@@ -7,11 +7,25 @@ image:
...
+
 service:
-  name: cluster-controller
+  # controller
   type: ClusterIP
   externalPort: 80
   internalPort: 80
+
+  # CRD
+  name: krakenclusters.samsung-cnct.com
```
[the remark this answers is not in the document]
Comment (leahnp) [reply charts/cluster-controller/values.yaml]: Great idea - I just altered the chart, I looked at istio and cert-manager mainly.

**Comment by leahnp** [comment]
@alejandroEsc thanks for the feedback, I used istio and cert-manager charts as reference, let me know if this is looking better!

File: charts/cluster-controller/templates/kraken-cluster.yaml [hunk 2]
```diff
@@ -0,0 +1,82 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+
+metadata:
+  name: krakenclusters.samsung-cnct.com
```
Comment (alejandroEsc) [inline charts/cluster-controller/templates/kraken-cluster.yaml]: samsung.cnct.com as per @oneilcin

File: charts/cluster-controller/values.yaml [hunk 3]
```diff
@@ -7,15 +7,15 @@ image:
   # TODO version this
   # tag: lastest
   pullPolicy: IfNotPresent
+
+krakenClusterShortNames: kc
```
Comment (alejandroEsc) [inline charts/cluster-controller/values.yaml]: wondering if we should or shouldnt make that optional, I would say not to, but yeah maybe in some clusters collisions can occur so maybe makes sense.

File: pipeline.yaml [hunk 4]
```diff
@@ -13,9 +13,9 @@ configs:
     release: cluster-controller
     test: 
       tests:
-        - image: bash:latest
-          script: scripts/stageTest.sh
-          shell: /bin/bash
+        # - image: bash:latest
```
Comment (alejandroEsc) [inline pipeline.yaml]: no need to comment things out. If they arent used please remove them.

File: charts/cluster-controller/values.yaml (hunk 3 above)
Comment (leahnp) [reply charts/cluster-controller/values.yaml]: I was just following the examples in the k8s repo

File: rootfs/cluster-controller/Dockerfile [hunk 5]
```diff
@@ -3,7 +3,9 @@ FROM golang:1.8
 WORKDIR /go/src/app
 COPY . .
 
-RUN go get -d -v ./...
-RUN go install -v ./...
+RUN curl https://glide.sh/get | sh
+
+RUN ls -ll
+RUN glide install
```
Comment (alejandroEsc) [inline rootfs/cluster-controller/Dockerfile]: one comment, we will remove glide reference onece the dep changes are committed. Let's make sure this is removed in that case.

**Comment by leahnp** [comment]
@alejandroEsc yes - thank you!

**Comment by alejandroEsc** [comment]
Lgtm.

**Pull request merged**